fulp_modules/features/antagonists/infiltrators/albino_gorilla.dm

```diff
@@ -8,8 +8,8 @@
 /mob/living/simple_animal/hostile/gorilla/albino/Initialize(mapload)
 	var/datum/action/cooldown/mob_cooldown/charge/gorilla/tackle = new(src)
 	tackle.Grant(src)
-	var/datum/action/cooldown/spell/pointed/projectile/gorilla_dung/poop = new(src)
-	poop.Grant(src)
+	var/datum/action/cooldown/spell/conjure/banana/trap = new(src)
+	trap.Grant(src)
```

Comment:
soulless

Reply:
administration issue. use ur admin powers to convince the goverment  or ur just a hypocrite